src/ws/cockpitws.h

```diff
@@ -32,6 +32,7 @@ extern const gchar *cockpit_ws_ssh_program;
 extern const gchar *cockpit_ws_bridge_program;
 extern const gchar *cockpit_ws_known_hosts;
 extern const gchar *cockpit_ws_default_host_header;
+extern const gchar *cockpit_ws_default_protocol_header;
```

Comment:
If this is just used in one test, I would not export it in the header. I would just add this line to the relevant test file along with a comment saying where the variable is used.